test/integration/features/basic.feature

```diff
@@ -84,6 +84,58 @@ Feature: Basic
       172.30.1.1:5000
       """
 
+  Scenario: Deploy ruby-ex s2i image
+    Given Minishift has state "Running"
+     When executing "oc login --username=developer --password=developer" retrying 24 times with wait period of 5 seconds
+     Then stderr should be empty
+      And exitcode should equal 0
+      And stdout should contain
+      """
+      Login successful
+      """
+    Then executing "oc new-project rubys2i" retrying 3 times with wait period of 5 seconds
+     And stderr should be empty
+     And exitcode should equal 0
+     And stdout should contain
+     """
+     Now using project
+     """
+    Then executing "oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git" retrying 3 times with wait period of 5 seconds
+     And stderr should be empty
+     And exitcode should equal 0
+     And stdout should contain
+     """
+     Success
+     """
+    Then executing "oc logs -f bc/ruby-ex" retrying 18 times with wait period of 5 seconds
```

Comment:
-1, so here we have a point where we indeed have to do something. The problem is that _oc logs_ will return until the builder image is downloaded. So we used to indeed have to have some sort of loop here. However, there is a better way of doing this. What you want is to wait for the app to be deployed. There is a command for that _oc rollout status deploymentconfig NAME --watch_. See also issue #641